controller/controller.go

```diff
@@ -43,6 +43,10 @@ func ServeStaticFile(w http.ResponseWriter, r *http.Request) {
 	fs := http.FileServer(filesDir)
 
 	w.Header().Set("Access-Control-Allow-Origin", "*")
+	w.Header().Set("Content-Security-Policy", "trusted-types")
```

Comment:
Do we need to explicitly disable trusted types? We should be able to just leave it off and use the default ("no trusted types").

Reply:
^ to be clear, that's just on el_main.js and translateelement.css, not element.js. are we using/proxying these resources?

Reply:
@diracdeltas yes we proxy all requests including static files